packages/lib-react-components/src/ZooHeader/components/UserMenu/UserMenu.js

```diff
@@ -20,7 +20,7 @@ export default function UserMenu ({ signOut, user }) {
   // Support staging urls...
   const host = getHost()
 
-  const userDisplayName = <NarrowMenuNavListItem color='#B2B2B2' text={user.display_name} />
+  const userDisplayName = <NarrowMenuNavListItem color='#B2B2B2' text={user.display_name} overrideLang />
```

Comment:
Maybe allow `lang` as an optional prop here.
```suggestion
  const userDisplayName = <NarrowMenuNavListItem color='#B2B2B2' text={user.display_name} lang='en' />
```